migrations/inc/getConfig.js

```diff
@@ -0,0 +1,31 @@
+const developmentConfig = require('../../config/development');
```

Comment:
Why is this under `migrations/inc/getConfig.js` seems like a weird path. What's the current convention to where to put this file?

Reply:
I only put it here because it is used for migration only to read configuration.
Idk if there is convention... probably depends on a case ie I did saw this:
```
var myAppData = require('myapp/libs/data');
module.exports = function(deployer) {
```
@therevoltingx I can move this getter to `scripts/` directory if you would like it better?